cranelift/codegen/src/prelude.isle

```diff
@@ -43,9 +43,6 @@
 (decl value_type (Type) Value)
 (extern extractor infallible value_type value_type)
 
-(decl pure u32_add (u32 u32) u32)
-(extern constructor u32_add u32_add)
-
 ;; Pure/fallible constructor that tries to add two `u32`s, interpreted
 ;; as signed values, and fails to match on overflow.
 (decl pure partial s32_add_fallible (u32 u32) u32)
```

Comment:
I guess `s32_add_fallible` ought to move to the same place as the other add operations as well, but that doesn't have to happen in this PR.